sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusManagementAsyncClient.java

```diff
@@ -1135,6 +1267,31 @@
         }
     }
 
+    /**
+     * Gets whether an entity exists.
+     *
+     * @param getEntityOperation Operation to get information about entity. If {@link ResourceNotFoundException} is
+     *     thrown, then it is mapped to false.
+     * @param <T> Entity type.
+     *
+     * @return True if the entity exists, false otherwise.
+     */
+    <T> Mono<Response<Boolean>> entityExistsWithResponse(Mono<Response<T>> getEntityOperation) {
+        return getEntityOperation.map(response -> {
```

Comment:
somewhere we have `get` prefix  and other places we do not. Example `getTopicExistsWithResponse`  and `entityExistsWithResponse` . Is it intentional ? Could this be symmetric ?